opta/crash_reporter.py

```diff
@@ -79,5 +79,5 @@ def emit(self, record: LogRecord) -> None:
 CURRENT_CRASH_REPORTER = CrashReporter()
 crash_report_handler = ListHandler(CURRENT_CRASH_REPORTER.log_lines)
 crash_report_handler.setLevel(DEBUG)
-crash_report_handler.setFormatter(LogFormatMultiplexer())
+crash_report_handler.setFormatter(SensitiveFormatter())
```

Comment:
Did you mean to replace LogFormatMultiplexer with SensitiveFormatter? Looks like you've already added SensitiveFormatter to the LogFormatMultiplexer class itself.

Reply:
Yes, this is intended

Reply:
We weren't multiplexing anyting there so it's fine

Reply:
cool!